### PR TITLE
NGS2_WITNESS_empanelment_prep.R

### PR DESCRIPTION
**Pull request by C-Pablo-Diego-Rosell**
I recoded the religion variable into a more manageable set of categories for analysis purposes.

### CODE REVIEW

File: NGS2_WITNESS_empanelment_prep.R [hunk 1]
```diff
@@ -127,6 +127,32 @@ empanel[, EMPANEL_YESNO_VARS] <- apply(empanel[, EMPANEL_YESNO_VARS], 2,
...
+empanel$religion[empanel$Q23_religion=="Spiritism"] <- "Other religion"
+empanel$religion[empanel$Q23_religion=="Buddhism"] <- "Other religion"
+empanel$religion[empanel$Q23_religion=="Jewish"] <- "Other religion"
+empanel$religion[empanel$Q23_religion=="Primal-indigenous/African Traditional and Diasporic/Animist/Nature Worship/Paganism"] <- "Other religion"
+empanel$religion[empanel$Q23_religion=="Chinese Traditional Religion/Confucianism"] <- "Other religion"
+empanel$religion[empanel$Q23_religion=="Sikhism"] <- "Other religion"
+empanel$religion[empanel$Q23_religion=="Hinduism"] <- "Other religion"
+empanel$religion[empanel$Q23_religion=="Other (Write in:)"] <- "Other religion"
+
+empanel$religion[empanel$Q23_religion=="No Religion/Atheist/Agnostic"] <- "No Religion/Atheist/Agnostic"
```
Comment (mhoover) [inline NGS2_WITNESS_empanelment_prep.R:151]: This line isn't necessary, but not a deal breaker.

**Review by mhoover — APPROVED** [review]
LGTM, 👍

**Comment by uduwage** [comment]
Just a thought, Are we group/collapsing religions based on any prior research...? This is a sensitive subject and it might be good to do the grouping on based on pre-identified / publish information.

**Comment by C-Pablo-Diego-Rosell** [comment]
Thanks for noting, Anu. These groups  are pretty non-controversial.

Sent from my iPhone

On 19 Aug 2017, at 00:27, Anuradha Uduwage <notifications@github.com<mailto:notifications@github.com>> wrote:


Just a thought, Are we group/collapsing religions based on any prior research...? This is a sensitive subject and it might be good to do the grouping on based on pre-identified / publish information.

-

**Pull request merged**